backend/main.py

```diff
@@ -490,6 +494,6 @@ async def delete_match(match_id: str):
 
 
 if __name__ == "__main__":
-    import uvicorn
+    import uvicorn # type: ignore
```

Comment:
Neither should this. These mypy errors are there most probably because your local python env is not setup. (As the env is provided via a dockerfile).